docs/source/api/letsencrypt_dnsrecord.rst

```diff
@@ -0,0 +1,66 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-letsencrypt-dnsrecord:
+
+*************************
+``letsencrypt/dnsrecord``
+*************************
+
+.. versionadded:: 1.4
+
+``GET``
+========
+Gets DNS challenge records for Let's Encrypt DNS challenge for a specified fqdn.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+.. table:: Request Query Parameters
+
+	+------+----------+------------------------------------------------------------+
+	| Name | Required | Description                                                |
+	+======+==========+============================================================+
+	| fqdn | yes      | Return only DNS challenge records for the specified fqdn   |
+	+------+----------+------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/1.4/letsencrypt/dnsrecord?fqdn=_acme-challenge.demo1.example.com. HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+
+
+Response Structure
+------------------
+:fqdn:      The Fully Qualified Domain Name (FQDN) for the TXT record location to complete the DNS challenge
```

Comment:
> _"... Fully Qualified Domain Name (FQDN) ..."_

There's a text role for that, so that it appears on hover in HTML or expanded exactly once per section in PDF output:

```rst
The :abbr:`FQDN (Fully Qualified Domain Name)` for the TXT record ....
```

Reply:
done